src/main/java/com/example/demo/repository/MatchRepository.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.repository;
+
+import com.example.demo.CSV.CSVlibrary;
+import com.example.demo.model.Match;
+import com.example.demo.model.Team;
+import lombok.Value;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@Value
```

Comment:
I think this is redundant. If you want to provide access to matches you have getList() method. This way you can control it's implementation and add logic in future(for example not include completed matches, etc)